tests-e2e/cypress/integration/backstage/playbook_creation_spec.js

```diff
@@ -70,9 +73,36 @@ describe('playbook creation button', () => {
         // # Click 'Incident Collaboration Playbook'
         cy.findByText('Incident Collaboration Playbook').should('be.visible').click().wait(TIMEOUTS.TINY);
 
-        //Verify a new 'Incident Response Playbook' creation page is opened
+        // * Verify a new 'Incident Response Playbook' creation page is opened
         verifyPlaybookCreationPageOpened(url, playbookName);
     });
+
+    it('shows remove beside members when > 2 members', () => {
+        // # Open backstage
+        cy.openBackstage();
+
+        // # Click 'New Playbook' button
+        cy.findByText('Create a Playbook').should('be.visible').click().wait(TIMEOUTS.TINY);
+
+        cy.findByTestId('playbook-sidebar').within(() => {
+            cy.findAllByTestId('user-line').should('have.length', 1);
+            cy.findAllByTestId('user-line').eq(0).within(() => {
+                cy.get('a').should('not.exist');
+            });
+
+            cy.get('.profile-autocomplete__input > input')
+                .type('anne stone', {force: true, delay: 100}).wait(100)
+                .type('{enter}');
+
+            cy.findAllByTestId('user-line').should('have.length', 2);
+            cy.findAllByTestId('user-line').eq(0).within(() => {
+                cy.get('a').contains('Remove').should('exist');
+            });
+            cy.findAllByTestId('user-line').eq(1).within(() => {
+                cy.get('a').contains('Remove').should('exist');
+            });
+        });
```

Comment:
If we have the `open backstage` comment, we should have these ones :joy: 

```suggestion
        cy.findByTestId('playbook-sidebar').within(() => {
            // * Verify that there is no Remove link when there is one member
            cy.findAllByTestId('user-line').should('have.length', 1);
            cy.findAllByTestId('user-line').eq(0).within(() => {
                cy.get('a').should('not.exist');
            });
           
            // # Add a new user
            cy.get('.profile-autocomplete__input > input')
                .type('anne stone', {force: true, delay: 100}).wait(100)
                .type('{enter}');

            // * Verify that there is a Remove link when there is more than one member
            cy.findAllByTestId('user-line').should('have.length', 2);
            cy.findAllByTestId('user-line').eq(0).within(() => {
                cy.get('a').contains('Remove').should('exist');
            });
            cy.findAllByTestId('user-line').eq(1).within(() => {
                cy.get('a').contains('Remove').should('exist');
            });
        });
```